doc/api/vm.md

```diff
@@ -1061,7 +1061,8 @@ current V8 isolate, or the main context.
     right away to measure the memory.
     **Default:** `'default'`
 * Returns: {Promise} If the memory is successfully measured the promise will
-  resolve with an object containing information about the memory usage.
+  resolve with an object containing information about the memory usage, else it
+  will be rejected with `ERR_CONTEXT_NOT_INITIALIZED` error.
```

Comment:
Probably better to say it will be rejected with an `Error` with a `ERR_CONTEXT_NOT_INITIALIZED` `.code` property or similar? 

I guess it's fine though?

Reply:
Updated, thanks for the review.

Reply:
I don't think we need to specify the code property each time, most of the docs uses the wording used by @yashLadha AFAIK.